webui/react/src/components/kit/LineChart.tsx

```diff
@@ -326,12 +326,15 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
       ? Loadable.getOrElse([], propChartsProps)
       : propChartsProps;
     const isLoading = Loadable.isLoadable(propChartsProps) && Loadable.isLoading(propChartsProps);
-
     // X-Axis control
+
     const xAxisOptions = useMemo(() => {
       const xOpts = new Set<string>();
       chartsProps.forEach((chart) => {
-        chart.series.forEach((serie) => {
+        const series = Loadable.isLoadable(chart.series)
+          ? Loadable.getOrElse([], chart.series)
+          : chart.series;
```

Comment:
Shouldn't this just the same as `const series = Loadable.getOrElse([], chart.series)`?